python/paddle/fluid/clip.py

```diff
@@ -30,7 +30,6 @@
 
 __all__ = [
     'set_gradient_clip',
-    'ErrorClipByValue',
     'ClipGradByValue',
     'ClipGradByNorm',
     'ClipGradByGlobalNorm',
```

Comment:
the implementation of these functions and classes are migrated to `paddle/nn/clip.py`, so the list of \_\_all\_\_ should be clean and implementation here should be deleted.

Reply:
This file should & has been removed with your suggestion.